_pages/sponsors.html

```diff
@@ -49,6 +49,13 @@ <h2 class="text-center v-pad-bottom">{{ level }}</h2>
         </div>
         <div class="column medium-8">
           <h3 class="partner-name"><a href="{{ sponsor.url_target }}" rel="sponsored">{{ sponsor.name }}</a></h3>
+          {% if sponsor.extra_packages %}
+          <div class="partner-packages">
```

Comment:
Result in case there are multiples elements in extra_packages (i.e : `["diversity", "social events sponsors"]` ) :
<img width="996" alt="Capture d’écran 2023-05-15 à 21 13 39" src="https://github.com/djangocon/2023.djangocon.eu/assets/17890338/98ce2c8a-0f72-429c-98d7-943c0247b923">

Reply:
I would just make it repeat the word "sponsor" - so it says "diversity sponsor", "social event sponsor". When it's in a lozenge  each one is a distinct thing, it doesn't read as a sentence across them